app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :content, presence: true, length: { maximum: 5000 }
+  validates :user_id, presence: true
```

Comment:
- [ ] **[OPTONIAL]** `belongs_to` association already adds presence validation -- There is no need for validating post_id or user_id presence

Reply:
### Fixed!!, thank you for your hard work 👍